xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserProfileSheet.xml

```diff
@@ -145,9 +150,12 @@ $services.localization.render('platform.core.profile.category.profile.disabled')
       #end
       &lt;div class='userRecentChanges'&gt;
       #if ($xcontext.user == $doc.fullName)
-        &lt;h1&gt;$services.localization.render('platform.core.profile.section.activity')&lt;/h1&gt;
+        &lt;h2&gt;$services.localization.render('platform.core.profile.section.activity')&lt;/h2&gt;
       #else
-        &lt;h1&gt;$services.localization.render('platform.core.profile.section.activityof', [$xwiki.getUserName($doc.fullName, false)])&lt;/h1&gt;
+        &lt;h2&gt;
+         $services.localization.render('platform.core.profile.section.activityof',
+         [$xwiki.getUserName($doc.fullName, false)])
```

Comment:
This line needs to be indented one level more, since it's a fragment of the previous line of code.